src/main/java/com/rackspace/salus/resource_management/web/controller/ResourceApiController.java

```diff
@@ -193,4 +193,10 @@ public void delete(@PathVariable String tenantId,
   public PagedContent<ResourceDTO> getResourcesBySearchString(@PathVariable String tenantId, @RequestParam("q") String searchCriteria, Pageable page) {
     return PagedContent.fromPage(resourceManagement.getResourcesBySearchString(tenantId, searchCriteria, page));
   }
+
+  @DeleteMapping("/tenant/{tenantId}/")
```

Comment:
To avoid accidental use via public API, I suggest an admin qualified path that's purposely non-standard like:
```suggestion
  @DeleteMapping("/admin/delete-all/{tenantId}/")
```